rstudio/rstudio
# Add editor scrolling sensitivity preference

### Intent
Address #11578 

### Approach
Adds a preference to adjust the scroll sensitivity. Higher results in faster scrolling. The range is 1-200, default 100. The default is set to use the device pixel ratio as the modifier on the ACE default scroll speed.

Preference screenshot:
![image](https://user-images.githubusercontent.com/9591545/191815270-a12eef40-7598-4465-886b-93e097c5465e.png)


### Automated Tests
None

### QA Notes
Any tests that rely on scroll speed might be affected. The behaviour on Windows should be the same. Linux and MacOS might be different but I believe this change really affects those with high DPI displays. 

### Checklist
